sync2jira/downstream_issue.py

```diff
@@ -61,12 +61,18 @@
         FROM
             JIRA_DB.MARTS.JIRA_REMOTELINK AS rl
             INNER JOIN JIRA_DB.MARTS.JIRA_ISSUE AS ji ON ji.ID = rl.ISSUEID
-            AND rl.TITLE = {} AND rl.URL = {}
+            AND rl.TITLE = '{remote_link_title}' AND rl.URL = %s
     ) AS a
     LEFT JOIN JIRA_DB.MARTS.JIRA_PROJECT AS p on a.project_id = p.ID
 """
 
 
+# URL validation
+def validate_github_url(url):
+    pattern = r"^https://github\.com/[^/]+/[^/]+/(issues|pull)/\d+$"
+    return bool(re.match(pattern, url))
```

Comment:
A few recommendations, here: 
- use [`fullmatch()`](https://docs.python.org/3/library/re.html#re.fullmatch) rather than `match()`, which allows you to omit the two anchors.
- use a "docstring" instead of a "block comment"
- use a pre-compiled pattern, so that we don't have to recompile it on each call.

```suggestion
GH_URL_PATTERN = re.compile(r"https://github\.com/[^/]+/[^/]+/(issues|pull)/\d+")


def validate_github_url(url):
    """URL validation"""
    return bool(GH_URL_PATTERN.fullmatch(url))
```

This makes the function simple enough that you should consider omitting it and just in-lining the expression where you are currently calling it.